integrationTests/testInitializer.go

```diff
@@ -446,3 +450,361 @@ func mintAddressesFromSameShard(nodes []*TestProcessorNode, targetNodeIdx int, v
 		MintAddress(targetNode.AccntState, n.PkTxSignBytes, value)
 	}
 }
+
+// IncrementAndPrintRound increments the given variable, and prints the message for teh beginning of the round
+func IncrementAndPrintRound(round uint64) uint64 {
+	round++
+	fmt.Printf("#################################### ROUND %d BEGINS ####################################\n\n", round)
+
+	return round
+}
+
+// DeployScTx creates and sends a SC tx
+func DeployScTx(nodes []*TestProcessorNode, senderIdx int, scCode string) {
+	fmt.Println("Deploying SC...")
+	txDeploy := createTxDeploy(nodes[senderIdx], scCode)
+	nodes[senderIdx].SendTransaction(txDeploy)
+	fmt.Println("Delaying for disseminating the deploy tx...")
+	time.Sleep(time.Second)
```

Comment:
might add a variable for this time.Second (stepDelay or something). Valid for all other time.Sleep from the rest of the file.

Reply:
Done.